tests/unit/s2n_aes_sha_composite_test.c

```diff
@@ -59,20 +53,17 @@ int main(int argc, char **argv)
     uint8_t mac_key_sha256[32] = "server key sha256server key sha";
     uint8_t aes128_key[] = "123456789012345";
     uint8_t aes256_key[] = "1234567890123456789012345678901";
-    struct s2n_blob aes128 = {.data = aes128_key,.size = sizeof(aes128_key) };
-    struct s2n_blob aes256 = {.data = aes256_key,.size = sizeof(aes256_key) };
-    struct s2n_blob r = {.data = random_data, .size = sizeof(random_data)};
+    struct s2n_blob aes128 = { .data = aes128_key, .size = sizeof(aes128_key) };
+    struct s2n_blob aes256 = { .data = aes256_key, .size = sizeof(aes256_key) };
+    struct s2n_blob r = { .data = random_data, .size = sizeof(random_data) };
     /* Stores explicit IVs used in each test case to validate uniqueness. */
     uint8_t existing_explicit_ivs[S2N_DEFAULT_FRAGMENT_LENGTH + 2][S2N_TLS_MAX_IV_LEN];
 
     BEGIN_TEST();
     EXPECT_SUCCESS(s2n_disable_tls13_in_test());
 
     /* Skip test if we can't use the ciphers */
-    if (!s2n_aes128_sha.is_available()    ||
-        !s2n_aes256_sha.is_available()    ||
-        !s2n_aes128_sha256.is_available() ||
-        !s2n_aes256_sha256.is_available()) {
+    if (!s2n_aes128_sha.is_available() || !s2n_aes256_sha.is_available() || !s2n_aes128_sha256.is_available() || !s2n_aes256_sha256.is_available()) {
```

Comment:
This is pretty long now. Might need to be manually wrapped.

Reply:
Done.